doc/src/quickstart/index.rst

```diff
@@ -332,14 +305,14 @@ However, there is an issue with the above BLIF produced by ABC: the latches (ris
 
 Re-inserting clocks
 ^^^^^^^^^^^^^^^^^^^
-We will restore the clock information by running a script which will transfer that information from the original ODIN BLIF file (writing it to the new file ``blink.pre-vpr.blif``):
+We will restore the clock information by running a script which will transfer that information from the original Parmys BLIF file (writing it to the new file ``blink.pre-vpr.blif``):
```

Comment:
I think this is only needed because we directly ran ABC on the parmys output, and that should be stated (when run directly, ABC does not preserve clocks so we re-insert them in the re-optimized design with the restore_multiclock_latch.pl script).